stl/inc/yvals_core.h

```diff
@@ -855,6 +855,42 @@
 
 // next warning number: STL4027
 
+// P0619R4 remove deprecated features from C++20

```

Comment:
This should additionally be listed in a `// _HAS_CXX20 indirectly controls:` section at the top, like `N4190` is.

Reply:
Added it after the `// _HAS_CXX20 directly controls:` block